ftw/simplelayout/upgrades/20170428171243_install_iframefix/metadata.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<metadata>
+  <dependencies>
+    <dependency>profile-ftw.iframefix:default</dependency>
```

Comment:
Instead of adding a metadata.xml, I prefer to add self.setup_install_profile('profile-ftw.iframefix:default') to the upgrade step (before running self.install_upgrade_profile()).